style.css

```
@@ -12,6 +8,15 @@ h1 {
     margin: 0;
 }
 
+li {
+    list-style-type: none;
+    list-style: none;
+}
+
+.Fist-page {

```

Typo here impacting your styles

```suggestion
.First-page {

```